src/coreclr/vm/jitinterface.cpp

```diff
@@ -12131,6 +12131,26 @@ HRESULT CEEJitInfo::allocPgoInstrumentationBySchema(
         codeSize = m_ILHeader->GetCodeSize();
     }
 
+    if (m_pMethodBeingCompiled->IsVersionable())
+    {
+        CodeVersionManager* pCodeVersionManager = m_pMethodBeingCompiled->GetCodeVersionManager();
+        CodeVersionManager::LockHolder codeVersioningLockHolder;
+        ILCodeVersion ilVersion = pCodeVersionManager->GetActiveILCodeVersion(m_pMethodBeingCompiled);
+        NativeCodeVersion currentVersion = ilVersion.GetActiveNativeCodeVersion(m_pMethodBeingCompiled);
```

Comment:
The active code version may have changed meanwhile, so it may not represent the current code version being jitted. Could use something like this instead:

https://github.com/dotnet/runtime/blob/7b662a7e264552ea19a51a0135f2441bd5f141aa/src/coreclr/vm/jitinterface.cpp#L6681

Along with `config->GetCodeVersion()`.